### PR TITLE
TF-2116 Apply new design composer

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#2116 

## Dependent

- Need merged: 
    - https://github.com/linagora/html-editor-enhanced/pull/20
    - https://github.com/linagora/enough_html_editor/pull/22

### 1. COMPOSER

- Web:

https://github.com/linagora/tmail-flutter/assets/80730648/4268ec7f-dbca-4701-8d2c-e014be6070d5

- Mobile

https://github.com/linagora/tmail-flutter/assets/80730648/7d19ede1-089d-4158-9d73-d9cd9e60a18d

### 2.  EMAIL VIEW

- Web:

https://github.com/linagora/tmail-flutter/assets/80730648/fdb39a89-a836-47e2-84e0-4336a1e3d6d3

- Mobile:

https://github.com/linagora/tmail-flutter/assets/80730648/9af900fa-cbf5-4109-91cc-731e83b35a55

### CODE REVIEW

**Comment by chibenwa** [comment]
Can you provide screenshots?

**Comment by dab246** [comment]
> Can you provide screenshots?

@chibenwa  Please see description at the top

**Review by nqhhdev — APPROVED** [review]

File: lib/features/upload/presentation/model/upload_file_state.dart [hunk 1]
```diff
@@ -18,30 +18,37 @@ class UploadFileState with EquatableMixin {
   final int uploadingProgress;
   final Attachment? attachment;
   final CancelToken? cancelToken;
+  final bool fromFileShared;
```
Comment (hoangdat) [inline lib/features/upload/presentation/model/upload_file_state.dart:21]: can you explain a little bit?

**Review by hoangdat — APPROVED** [review]

File: lib/features/upload/presentation/model/upload_file_state.dart (hunk 1 above)
Comment (dab246) [reply lib/features/upload/presentation/model/upload_file_state.dart:21]: Before @manh did the inline image insert, always insert it at the end of the editor by calling the `moveCursorAtLastNode` function. So I added the variable `fromFileShared` to be able to insert the image inline at the cursor position. And only insert at the end when the image is shared from another application.

**Pull request merged**